packages/core-components/src/components/WarningPanel/WarningPanel.tsx

```diff
@@ -153,6 +155,8 @@ const capitalize = (s: string) => {
  *        the cause of the error.
  * @param children - Objects to provide context, such as a stack trace or detailed
  *        error reporting. Will be available inside an unfolded accordion.
+ * @param subtitle - A title for the heading. If not supplied, one will be constructed
```

Comment:
Don't think this change makes sense tbh, since the `subTitle` in the implementation is really just a refinement of the `title`. There's no title + subtitle combo